src/js_crypto/bigint-helpers.ts

```diff
@@ -0,0 +1,93 @@
+export { changeBase, bytesToBigInt, bigIntToBytes };
```

Comment:
this new file collects from helpers which were scattered through the code base - nothing here is new